codecov.yml

```diff
@@ -1,9 +1,16 @@
 #
-# For documentation see https://codecov.io/gh/twisted/twisted/settings/yaml
+# For documentation: https://docs.codecov.io/docs/codecovyml-reference
+# Twisted settings: https://codecov.io/gh/twisted/twisted/settings/yaml
 #
 # We want 100% coverage for new patches to make sure we are always increasing
 # the coverage.
 #
+codecov:
+  require_ci_to_pass: yes
+  notify:
+    after_n_builds: 7
+    wait_for_ci: yes
```

Comment:
Here I am trying to see if I can configure codecov.io to wait a bit more before sending the coverage reports so that we don't have transient red checks

Reply:
I don't think the existing behavior is bad, necessarily, it's just confusing for people who (like me) assume that a check is complete once it shows either a check or X.

Reply:
In my experience these two options don't actually work.  I never got to the bottom of why and codecov.io declined to assist.

Reply:
I will try to find some time and report this issue via the internal / paid support codecov channel.

It is true that codecov are not helping too much with open source projects... so we are on a free ride here.

But I see if I can reproduce it for my private project and try to get help via that project.